Sprint-2/extend/format-time.js

```diff
@@ -22,3 +22,25 @@ console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
 );
+
+const currentOutput3 = formatAs12HourClock("15:00");
+const targetOutput3 = "3:00 pm";
+console.assert(
+  currentOutput3 === targetOutput3,
+  `current output: ${currentOutput3}, target output: ${targetOutput3}`
+);
+
+
+const currentOutput4 = formatAs12HourClock("00:00");
+const targetOutput4 = "12:00 pm";
+console.assert(
+  currentOutput4 === targetOutput4,
+  `current output: ${currentOutput4}, target output: ${targetOutput4}`
+);
+
+const currentOutput5 = formatAs12HourClock("00:00");
```

Comment:
This assertion is the same as the one above?

Reply:
Yes it seems like it, I did not notice it, I will correct my error and put my test cases on a test file.